<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QMainWindow" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="header" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000022;
border: none;</string>
    </property>
    <widget class="QPushButton" name="a_cube">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>140</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>Cube</string>
     </property>
    </widget>
    <widget class="QPushButton" name="a_waterripple">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>140</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>WaterRipple</string>
     </property>
    </widget>
    <widget class="QPushButton" name="a_kinematics">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>140</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>InvKinematic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="a_fireworks">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>140</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>Fireworks</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="header_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>250</width>
      <height>550</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000022;
border: none;</string>
    </property>
   </widget>
   <widget class="QWidget" name="header_3" native="true">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>200</y>
      <width>250</width>
      <height>550</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000022;
border: none;</string>
    </property>
   </widget>
   <widget class="QWidget" name="header_4" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>700</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000022;
border: none;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
